mods/d2k/rules/world.yaml

```diff
@@ -35,6 +35,10 @@
 		Name: Corrino
 		InternalName: corrino
 		Selectable: false
+	Faction@Smuggler:
```

Comment:
Nope, mercenaries were yet another faction that copied one of the base factions for everything except their custom heavy factory.